app/javascript/react/components/App.js

```diff
@@ -1,7 +1,19 @@
-import React from 'react'
+import React, { Fragment } from "react";
 
```

Comment:
lets remove the empty line 2.  While a space between the import statements concerning 3rd party libraries and files that you have made (line 5) are good, we dont need extra spaces within a group of import statements